frontend/src/content/PostList.tsx

```diff
@@ -71,7 +71,7 @@ const PostList = ({ feedType, profile, refreshTrigger, network, showReplyInput }
   useEffect(() => {
     const getOlder = refreshTrigger === priorTrigger;
     fetchData(getOlder);
-  }, [feedType, profile, refreshTrigger, priorTrigger, network]);
```

Comment:
The network is defaulted to `testnet` before login, then it changes to `dev`.  Is that the expected behavior or do we want to change that?

I don't mind this fix because I can't figure out a way to create the scenario where we're logged in and the network changes. If you have any pushback, lmk.

Reply:
Agreed, I don't think there's a valid reason to remain logged in and change networks.